cmd/kubectl-gadget/version.go

```diff
@@ -33,32 +46,70 @@ var versionCmd = &cobra.Command{
 	Short:        "Show version",
 	SilenceUsage: true,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		fmt.Printf("Client version: v%s\n", version.Version())
+		outputFormat, err := cmd.Flags().GetString("output")
+		if err != nil {
+			return fmt.Errorf("error getting output format: %w", err)
```

Comment:
nit: do not use the "error ..." prefix on error messages.

```suggestion
			return fmt.Errorf("getting output format: %w", err)
```